modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1545,6 +1545,8 @@ public ObjectEntry updateObjectEntry(
 
 		objectEntryPersistence.clearCache(SetUtil.fromArray(objectEntryId));
 
+		ObjectEntry originalObjectEntry = objectEntry;
+
 		objectEntry = objectEntryPersistence.findByPrimaryKey(objectEntryId);
```

Comment:
@G-Sa can you try to do something similar to this [code](https://github.com/carolmariaabb/liferay-portal/blob/344689f54b5ede48c9610059aed6ca63f777ba41/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java#L1316-L1319)?